Etabs_Adapter/Create/Load.cs

```diff
@@ -42,14 +42,13 @@ public bool CreateObject(Loadcase loadcase)
         {
             eLoadPatternType patternType = loadcase.Nature.ToCSI();
 
-            if (m_model.LoadPatterns.Add(loadcase.Name, patternType) == 0)
-            {
-                loadcase.CustomData[AdapterId] = loadcase.Name;
-            }
-            else
-            {
-                CreateElementError("Loadcase", loadcase.Name);
-            }
+            double selfWeight = 0;
+            if (loadcase.Nature == LoadNature.Dead)
```

Comment:
Are we sure we want to do this? This will make this adapter behave quite differently to the other ones. Have solved this using the GravityLoad before for Etabs, same as is done in the other adapters.

Reply:
Not sure at all! This was in the ETABS toolkit when I copied the code into SAP, so I figured it was good.
I don't know about GravityLoad- I'll read up and see if I can implement quickly (if it's not done already)

Reply:
After reviewing, I've removed the selfweight bit from Loadcase.

I also noticed a naming inconsistency: Etabs_Adapter vs. ETABS_Engine. Is this as simple a fix as I think it is? Will anything be impacted by changing the name of Etabs_Adapter to ETABS_Adapter?